test/integration/features/provision_earlier_version.feature

```diff
@@ -0,0 +1,51 @@
+@minishift-only
+Feature: Provision an older major release
+  As a user I can provision an older major version of minishift
+  Scenario: Starting Minishift with v1.5.1
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start --openshift-version v1.5.1" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: Provisioned Minishift have openshift v1.5.1
+     When executing "minishift openshift version" succeeds
+     Then stderr should be empty
+     And exitcode should equal "0"
+     And stdout should contain
+     """
+     openshift v1.5.1
+     """
+
+  Scenario: OpenShift is ready after startup
+    After startup of Minishift OpenShift instance should respond correctly on its html endpoints
+    and OpenShift web console should be accessible.
+    Given Minishift has state "Running"
+     Then status code of HTTP request to "OpenShift" at "/healthz" is equal to "200"
+      And body of HTTP request to "OpenShift" at "/healthz" contains "ok"
+      And status code of HTTP request to "OpenShift" at "/healthz/ready" is equal to "200"
+      And body of HTTP request to "OpenShift" at "/healthz/ready" contains "ok"
+      And status code of HTTP request to "OpenShift" at "/console" is equal to "200"
+      And body of HTTP request to "OpenShift" at "/console" contains "<title>OpenShift Web Console</title>"
+
+  Scenario: User has a pre-configured set of persistent volumes
```

Comment:
We need to remove this scenario or enable anyuid. There is no guarantee that this tests passed otherwise (have failures on locally). See https://github.com/minishift/minishift/blob/master/pkg/minishift/clusterup/clusterup.go#L214